drivers/usb/device/usb_dc_stm32.c

```diff
@@ -89,6 +89,12 @@ static const struct gpio_dt_spec ulpi_reset =
 	GPIO_DT_SPEC_GET_OR(DT_PHANDLE(DT_INST(0, st_stm32_otghs), phys), reset_gpios, {0});
 #endif
 
+#if USB_OTG_HS_ULPI_PHY || \
+DT_HAS_COMPAT_STATUS_OKAY(st_stm32u5_otghs_phy) || \
+USB_OTG_HS_EMB_PHYC
```

Comment:
Indent

Reply:
Actually here also `#if USB_OTG_HS_ULPI_PHY || USB_OTG_HS_EMB_PHYC` should be enough.